src/lib.rs

```diff
@@ -1,62 +1,65 @@
 use pyo3::prelude::*;
-use pyo3::types::PyBytes;
+mod inner {
```

Comment:
Question: What is the namespace issue you were seeing? I've not seen any issues with that

Reply:
When running `cargo build`:
```
error[E0425]: cannot find function `hash_seeded` in module `seahash`
  --> src\lib.rs:13:34
   |
13 |     py.allow_threads(|| seahash::hash_seeded(buf, a, b, c, d))
   |                                  ^^^^^^^^^^^ not found in `seahash`
   |
```

Reply:
Maybe the pyo3 pymodule macro changed the way it scopes the function or the way it mangles its name.

Reply:
It's possible, for sure. It does make things a little clearer anyway.